ci/spack.yaml

```diff
@@ -8,6 +8,7 @@ spack:
   - netcdf-fortran@4.5.3
   - w3emc@2.9.2
   - gsi-ncdiag@1.0.0
+  - bacio@2.4.1
```

Comment:
I think you want to move this above the `w3emc` line.  I don't think it matters, but since it is a dependency, it makes sense.

Reply:
Ok, I'll make that change.  Thanks.